### DISCUSSION TITLE
[Feature request] Colours that are light and dark theme compatible

### DISCUSSION THREAD

**Discussion by adam-nielsen** [grafana/grafana#37575] [Dashboards, Panels & Visualizations]
Currently there are few graph metric colours that work with both the light and dark theme.

There is a colour called "text colour", but this gets translated into black in the light theme (which is then invisible in the dark theme), and if you select it in the dark theme it gets saved as white, which is then invisible in the light theme.

If you have some users using the light theme, and some using the dark theme, it can be difficult to select colours that stand out in both themes.

Looking at [this CSS example](https://css-tricks.com/reverse-text-color-mix-blend-mode/) it's possible to use CSS to invert colours based on the background colour, so adding something like this to the colour selection would make it easier to produce graphs that work well on both light and dark themes.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
